scripts/cjdns-iptunnel/install

```diff
@@ -5,5 +5,16 @@ set -e
 BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 # Install scripts that run after cjdns tun0 is configured
-sudo cp "$BASE_DIR/50-cjdns.rules" /etc/udev/rules.d/50-cjdns.rules
 sudo cp "$BASE_DIR/cjdns-setup" /usr/local/sbin/cjdns-setup
+
+# Update service to start script on cjdns start
+if [ -f /lib/systemd/system/cjdns.service ]; then
+    sudo sed /ExecStartPost/d /lib/systemd/system/cjdns.service
+    sed -i s#Restart=always#Restart=always\\nExecStartPost=/usr/local/sbin/cjdns-setup# /lib/systemd/system/cjdns.service
+fi
+if [ -f /etc/systemd/system/cjdns.service ]; then 
```

Comment:
Why do we have this in two different dir in the fist place?? Can we fix the real problem?

Reply:
@benhylau 
Its not so much as problem but how the install happend

`deb` packages install in `/lib/systemd/system`  because its "part of the system"
`compiled` installs in /etc/systemd/system` because its "user created" script

So it depends if you installed the deb or compiled.

Reply:
Made issue to do this #262